runtime/doc/api.txt

```diff
@@ -638,7 +638,7 @@ nvim_set_client_info({name}, {version}, {type}, {methods},
                                   Clients might define their own keys, but the
                                   following are suggested:   "website" Website
                                   of client (for instance github repository)
-                                  "license" Informal descripton of the
+                                  "license" Informal description of the
```

Comment:
This is autogenerated from `src/nvim/api/vim.c` (doxygen comment above `nvim_set_client_info`), please change it there instead.